lib/theme/app_dimensions.dart

```diff
@@ -27,6 +27,7 @@ class AppDimensions {
   static const answerSmileyTextSize = 25.0;
   static const answerEmojiHeight = 70.0;
   static const answerEmojiTextSize = 25.0;
+  static const answerTextAreaHeight = 168.0;
```

Comment:
(Optional) For consistency with the widget name
```suggestion
  static const answerTextareaHeight = 168.0;
```